command/loadbot/execution.go

```diff
@@ -183,12 +222,18 @@ func (l *Loadbot) Run() error {
 	ticker := time.NewTicker(1 * time.Second / time.Duration(l.cfg.TPS))
 	defer ticker.Stop()
 
-	var wg sync.WaitGroup
-
-	receiptTimeout := calcMaxTimeout(l.cfg.Count, l.cfg.TPS)
+	// max-wait by default is 2 min.
+	receiptTimeout := time.Duration(l.cfg.MaxWait) * time.Minute
 
 	startTime := time.Now()
 
+	// deploy contracts
+	if err := l.deployContract(grpcClient, jsonClient, receiptTimeout); err != nil {
+		return fmt.Errorf("unable to deploy smart contract, %w", err)
+	}
```

Comment:
Looking at the code it seems like this is a necessary step (contract deployment), when in fact it's only necessary for a subset of modes.

Please consider wrapping this somehow so it's not confusing 🙏

Reply:
@ZeljkoBenovic regarding this, create a simple helper `isTokenTransferMode()` so that you only call `deployContract` if that is the case.

Reply:
Added isTokenTransferMode 65207facceda48c54b7e2ec14a7f98fe46b37d00